### PR TITLE
docs: update faq [skip chromatic]

### LINKED ISSUE #1605: docs: 📚 update FAQ
## Description
Update the FAQ from => https://www.figma.com/design/ID5rwf5MVDgDEpOCMp5rYN/SDS-FAQs?node-id=2218-97&node-type=frame&t=44CmUY8lzd04LO8k-11

## Where to put the documentation
Storybook
## DoR
- [x] Item has business value
- [x] Item has been estimated by the team
- [x] Item is clear and well-defined
- [x] Item dependencies have been identified

## DoD
- [ ] Documentation has been created/updated (if applicable)
- [ ] Migration Guide has been created/updated (if applicable)
- [ ] Relevant stories (Features, A11y) are created/updated
- [ ] Implementation works successfully on `feature` branch

### PR DESCRIPTION
**Pull request by karlbaumhauer**
## Description:
- updates faq
- re-formats the texts
- adds new faq

closes #1605 
## Definition of Reviewable:
- [x] Documentation is created/updated
- [x] relevant tickets are linked

### CODE REVIEW

**Comment by karlbaumhauer** [comment]
@miguelxnxn pls have a look into it. In case you have questions or suggestions, pls add inline comments in the review section here on github.

The only thing I did not change was the spacing in list elements => figma comment

File: packages/components/src/docs/General/11-FAQ.mdx [hunk 1]
```diff
@@ -151,34 +135,23 @@ import 'https://solid-design-system.fe.union-investment.de/x.x.x/components/es/a
...
         </span>
       </sd-accordion>
-
       <sd-accordion>
         <h4 class="text-base" slot="summary">
           Does the design system support TypeScript and work with React, Vue, and Angular?
         </h4>
         <span class="sd-prose">
-          The Solid Design System is designed to be framework-agnostic, meaning it can be integrated with various frameworks,
-          including React, Vue, Angular and <a target="_blank" href="https://custom-elements-everywhere.com/">many more</a>. Additionally, TypeScript
```
Comment (miguelxnxn) [inline packages/components/src/docs/General/11-FAQ.mdx:165]: @karlbaumhauer  Providing an external link here to "custom-elements-everywhere.com" under the linktext "many more" can be ambiguous.
The linktext should describe the target, e.g.: "including React, Vu, and Angular through the use of _custom elements_."

Comment (karlbaumhauer) [reply packages/components/src/docs/General/11-FAQ.mdx:165]: @miguelxnxn the point here is, that it actually inlcudes **many more** frameworks than just React, Vue and Angular. Thats why I added this.

Comment (miguelxnxn) [reply packages/components/src/docs/General/11-FAQ.mdx:165]: Could we then say:
"including React, Vu, Angular and many more through the use of _custom elements_."? This way, the link text informs about its target.

Comment (karlbaumhauer) [reply packages/components/src/docs/General/11-FAQ.mdx]: updated

**Review by miguelxnxn — APPROVED** [review]
Styling and content have been updated correctly.

**Review by miguelxnxn — APPROVED** [review]
Link to framework compatibility and sentence construction has been updated correctly.

**Review by miguelxnxn — APPROVED** [review]

**Review by miguelxnxn — APPROVED** [review]

**Review by miguelxnxn — DISMISSED** [review]

**Comment by miguelxnxn** [comment]
@karlbaumhauer I reviewed the changes, everything looks good. Could you please merge?

File: packages/docs/src/stories/docs/general/11-FAQ.mdx
[the remark this answers is not in the document]
Comment (karlbaumhauer) [reply packages/docs/src/stories/docs/general/11-FAQ.mdx:1]: done

**Pull request merged**